File: app/src/main/java/com/utair/presentation/utils/DebugUtils.java
package com.utair.presentation.utils;

import android.widget.Toast;

import com.utair.UTairApplication;

public class DebugUtils {

    private final static boolean ERRORS_ENABLED = true;

    public static void showDebugErrorMessage(Throwable throwable) {
        if (ERRORS_ENABLED) {
            throwable.printStackTrace();
            Toast.makeText(UTairApplication.context, throwable.getMessage(), Toast.LENGTH_SHORT).show();
        }
    }

}

File: app/src/main/java/com/utair/presentation/utils/DebugUtils.kt
package com.utair.presentation.utils

import android.widget.Toast
import com.utair.UTairApplication

object DebugUtils {

    private const val ERRORS_ENABLED = true

    fun showDebugErrorMessage(throwable: Throwable) {
        if (ERRORS_ENABLED) {
            throwable.printStackTrace()
            Toast.makeText(UTairApplication.context, throwable.message, Toast.LENGTH_SHORT).show()
        }
    }

}